README.md

```diff
@@ -91,6 +108,7 @@ A temporary workaround consists of [mocking global `console.error` to ignore the
 
 - Consider reusing Next.js code parts (not only types)
 - Consider supporting Next.js `trailingSlash` option
+- Don't provide `getInitialProps` with `ctx` object when called client-side
```

Comment:
This can be removed 🙂 See https://github.com/toomuchdesign/next-page-tester/pull/45#issuecomment-739793391

Reply:
🤟